elife/uwsgi.sls

```diff
@@ -18,8 +18,8 @@ uwsgi-pkg:
 {% else %}
 # apps should be installing and using their own version of uwsgi
 uwsgi-pkg:
-    pip.installed:
-        - name: uwsgi >= 2.0.8
+    cmd.run:
+        - name: pip install "uwsgi>=2.0.8"
```

Comment:
we should figure out what projects are still using the global uwsgi and update them.

Reply:
some that have been moved to Docker containers naturally had to use their own `uwsgi` in the virtualenv